product_analytic_sale/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,10 @@
+This module allows to compute the analytic distribution on sale order line level
+from the one stored either on product category either on product level
+(see product_analytic module).
+
+Developers
+==========
+
```

Comment:
```suggestion
```

Reply:
Why ? This is a technical section for developers.

Reply:
Because we must not add top level section titles in README fragments as this breaks the rendered document.

Reply:
aaaah ok. Is this better ?

Reply:
Nope. If you really want a subsection, then use `~`, but really there is no reason to create a subsection just for one line.